### DISCUSSION TITLE
Data types

### DISCUSSION THREAD

**Discussion by gmartinsribeiro** [cleanlab/cleanlab#124] [Ideas]
Hey there!

Love what you're doing here!
Quick question: are you focused on images only? Will you have support for other data types?

Thanks

**Comment by anishathalye** [comment]
Cleanlab is agnostic to data modality (and also model-agnostic). It can be and has been used for other data types, including text, tabular, and audio datasets. See https://labelerrors.com/ (and implementation using cleanlab at https://github.com/cleanlab/label-errors) for some examples.
